kubernetes/package.dhall

```diff
@@ -0,0 +1,5 @@
+{ Duration = ./duration.dhall
+, Image = ./image.dhall
+, Compute = ./compute.dhall
+, Space = ./space.dhall
+}
```

Comment:
I think that if we're going to have all of the file start with a lower case letter, then the package names should also start with a lower case letter. It should match the name of the file exactly.

Reply:
I tend to agree, but we need to come up with convention for all of our open-source projects.

To me Upper case makes more sense (both name and files), but it means that the map should also be changed accordingly. Thoughts?

Reply:
The original convention was to start with an upper case letter for a Type and start with a lower case letter for a record.
Basically all of our open-source work is "smart types" and not so much records / data.
So if this is the case then `duration.dhall` should be `Duration.dhall` etc.
keep `package.dhall` with a lower-case p since it is a record of types.